benchalerts/tests/unit_tests/expected_md/comment_noresults.md

```diff
@@ -0,0 +1,5 @@
+Conbench analyzed the 1 benchmark run on commit `abc`.
```

Comment:
Style/language questions which we can resolve/conclude later.

- 'analyzed 1 run' vs 'analyzed the 1 run' -- as a reader, I don't know what _the_ refers to, and I can live w/o it
- 'one' vs '1'. In scientific publishing, I had once learned that we want to use words for numbers smaller than 11 or ten :D (see? haha). This is not a rule, just a guideline, but here I think the guideline applies, aesthetically.

Reply:
From here, we can refine further. "run" is a tough concept, as we find out.

> Conbench analyzed one run of benchmarks on ...

But yeah. For later probably.